x-pack/plugins/enterprise_search/server/routes/enterprise_search/engines.ts

```diff
@@ -4,11 +4,17 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+import { SearchResponse } from '@elastic/elasticsearch/lib/api/types';
 import { schema } from '@kbn/config-schema';
 
-import { EnterpriseSearchEngineDetails } from '../../../common/types/engines';
+import { AcknowledgeResponse } from '@kbn/core-saved-objects-migration-server-internal/src/actions';
```

Comment:
I believe this is the wrong type import. Should have one in `@elastic/elasticsearch/lib/api/types`

Reply:
I think AcknowledgeResponse is not defined in `@elastic/elasticsearch/lib/api/types`

https://github.com/search?q=repo%3Aelastic%2Fkibana%20AcknowledgeResponse&type=code

Reply:
Ah!!  please ignore my previous comment. Found a similar response : `AcknowledgedResponseBase` in `@elastic/elasticsearch/lib/api/types`